File: src/remove_vowels.py
def remove_vowels(s: str) -> str:
    """
    Removes all vowels from the input string.

    Author: Your Name

    Args:
        s (str): Input string.

    Returns:
        str: String with all vowels removed.
    """
    vowels = "aeiouAEIOU"
    return "".join(char for char in s if char not in vowels)

File: src/remove_vowels.rs
/// Removes all vowels from the input string.
/// Author: Your Name
pub fn remove_vowels(s: &str) -> String {
    let vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
    s.chars()
        .filter(|c| !vowels.contains(c))
        .collect()
}
